dev-ml/parsexp/parsexp-0.11.0.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit opam
+
+DESCRIPTION="S-expression parsing library"
+HOMEPAGE="https://github.com/janestreet/parsexp"
+SRC_URI="https://github.com/janestreet/parsexp/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~ppc ~x86"
```

Comment:
You can't add ppc on new ebuilds. It needs to be tested with proper arch team member.